src/vscode-bicep/src/commands/build.ts

```diff
@@ -18,6 +18,14 @@ export class BuildCommand implements Command {
       return;
     }
 
+    if (documentUri.scheme === "output") {
```

Comment:
After the changes you made to package.json, what steps do you need to take to repro this?

Reply:
1. Open a .bicep file to activate the Bicep extension.
2. Open the output panel and focus it.
3. Run `bicep: Build` or `bicep: Open Visualizer` using the VS Code command palette.

Reply:
Ah - I missed that the active editor behind the output panel has to be a `.bicep` one. That's... odd!

Reply:
Yeah...somehow "editorLangId == bicep" does not work here (seems to be a part of the bug).